src/AzureClient/Visualization/HistogramEncoders.cs

```diff
@@ -72,7 +72,7 @@ public class HistogramToHtmlEncoder : IResultEncoder
                         return FormattableString.Invariant($@"
                             <tr>
                                 <td style=""{columnStyle}"">{result}</td>
-                                <td style=""{columnStyle}"">{frequency}</td>
```

Comment:
As a minor suggestion, we got feature requests for DumpMachine histograms in the past for configurable precisions; should this be configurable in the same way as @Reemeela2121 has been working on for DumpMachine?

Reply:
> As a minor suggestion, we got feature requests for DumpMachine histograms in the past for configurable precisions; should this be configurable in the same way as @Reemeela2121 has been working on for DumpMachine?

Yes, this makes sense. I'm closing this PR in favor of closer integration with the work that @Reemeela2121 has done. I've opened https://github.com/microsoft/iqsharp/issues/269 to track that.